paddlecloud/paddlejob/views.py

```diff
@@ -53,7 +55,8 @@ def post(self, request, format=None):
             psmemory = obj.get("psmemory", "1Gi"),
             topology = obj["topology"],
             gpu = obj.get("gpu", 0),
-            image = obj.get("image", "yancey1989/paddlecloud-job"),
+            image = obj.get("image", settings.JOB_DOCKER_IMAGE["image"]),
+            registry_secret = registry_secret,
```

Comment:
Yep, if there is no registry_secret be configured: [L46](https://github.com/PaddlePaddle/cloud/pull/67/files#diff-d0ac637065e625dbed7cba889a2aaccbR46),  and the Kubernetes job won't use registry secret.